UnitTests/GitUI.Tests/ResourcesTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Buffers.Binary;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.Globalization;
+using System.IO;
+using System.Resources;
+using System.Text;
+using FluentAssertions;
+using GitUI.Properties;
+using NUnit.Framework;
+
+namespace GitUITests
+{
+    [TestFixture]
+    public class ResourcesTests
+    {
+        [Test]
+        public void PngResources_ShouldNotContainDPIInformation()
+        {
+            // arrange
+            ResourceSet resourceSet = Images.ResourceManager.GetResourceSet(CultureInfo.InvariantCulture, true, false);
```

Comment:
`ResourceSet` is [disposable](https://docs.microsoft.com/en-us/dotnet/api/system.resources.resourceset?view=net-6.0)
```suggestion
            using ResourceSet resourceSet = Images.ResourceManager.GetResourceSet(CultureInfo.InvariantCulture, true, false);
```

Also please add argument names for `true` and `false`.

Reply:
Actually it should not be disposed, as otherwise a lot of tests are failing. Apparently, you retrieve a global resource, but not own it 😕 
Dropped a comment on that